stable/metrics/values.yaml

```diff
@@ -34,6 +34,13 @@ conf:
   redisPassword: TBD
   dogstatsdApiKey: TBD
 
+sqsExporter:
+  enabled: false
+  image:
+    repository: jesusfcr/sqs-exporter
+    tag: latest
```

Comment:
It would be possible to use a "fixed" version?
Perhaps new TAG version or even a specific commit version.
For internal dependencies, using latest, even if I think is not best practices is acceptable, for "external" dependencies is a bit tricky. WDYT?